docfx.json

```diff
@@ -105,6 +118,18 @@
       },
       "ROBOTS": {
         "api/**.yml": "NOINDEX, NOFOLLOW"
+      },
+      "ms.author": {
+        "_csharplang/spec/*.md": "wiwagn"
+      },
+      "topic": {
+        "_csharplang/spec/*.md": "reference"
```

Comment:
shouldn't this be ms.topic instead? line 113 already defines those for api topics. also one of my pet peeves of the current metadata page is that it doesn't show all the accepted values for reference. language-reference is one of those if you wanted to tag these with that.